src/main/java/org/zendesk/client/v2/model/TimeZone.java

```diff
@@ -0,0 +1,85 @@
+package org.zendesk.client.v2.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+/**
+ * @author nkharabaruk
+ * @since 22/06/2023 22:00
+ */
+public class TimeZone {
+
+    private String translatedName;
+    private String name;
+    private String ianaName;
+    private String url;
```

Comment:
I don't see this field in the documentation, but maybe I missed it, it's a bit of a pain to navigate... Do you have a link I can check?

Reply:
The API call returns timezones with the`url` field. 
I also don't see it in the documentation: https://developer.zendesk.com/api-reference/apps/apps-support-api/all_locations/#time-zone-object
✔️ removed